src/main/java/seedu/flashnotes/logic/parser/FlashNotesParser.java

```diff
@@ -60,12 +60,15 @@ public Command parseCommand(String userInput, boolean isInDeck, String deckName)
         final String arguments = matcher.group("arguments");
 
         if (this.isReviewMode) {
+            assert !isInDeck: "Program should not be in card mode";
```

Comment:
Should it be assert isInDeck instead? Since the user needs to be in a deck before going into review mode (like the isInDeck still remains true if the reviewMode is true

Reply:
Yes, you're right thanks!